README.md

```diff
@@ -114,6 +114,13 @@ Once the installation completed the command wil result in following output:
 "complete"
 ```
 
+### 6. Deploy a custom image of the operator
```

Comment:
I would say this isn't step 6. of the install but rather an alternative to step 4 of running the operator.

Reply:
Yep, it make more sense actually.